config/config_dev_kai.yaml

```diff
@@ -0,0 +1,67 @@
+log_directory: /report_service/logs/final
+profile_name: saml-pub
+s3_bucket: pds-web-analytics-cloud
+s3_logdir: logs_dev_kai
+subdirs:
+  atm:
+    atm-apache-http:
+      include:
+        - "atm-atmos-apache.2024*.txt"
+  en:
+    en-http-pds-proxy:
+      include:
+        - "*.2024*.txt.gz"
+    en-http-pdscloud-prod1:
+      include:
+        - "*.2024*.txt.gz"
+    en-http-pdscloud-prod2:
+      include:
+        - "*.2024*.txt.gz"
+  geo:
+    geo-ode:
+      include:
+        - "u_ex24*"
+    geo-ode-cart:
+      include:
+        - "u_ex24*"
+    geo-pds:
+      include:
+        - "u_ex24*"
+    geo-speclib:
+      include:
+        - "u_ex24*"
+    geo-va:
+      include:
+        - "u_ex24*"
+  img:
+    img-pdsimage-http:
+      include:
+        - "*.2024*"
+    img-pdsimg1-http:
+      include:
+        - "*.2024*"
```

Comment:
With their new machines, this log filename has changed such that this pattern doesn't work anymore. It should be changed to "*-2024*".